utils/src/main/java/software/amazon/awssdk/utils/FunctionalUtils.java

```diff
@@ -20,7 +20,9 @@
 import java.util.function.Consumer;
 import java.util.function.Function;
 import java.util.function.Supplier;
+import software.amazon.awssdk.annotations.SdkProtectedApi;
 
+@SdkProtectedApi
```

Comment:
Public?

Reply:
Before GA, sure. It's not ready to be public right now, though.